components/layout_2020/flow/text_run.rs

```diff
@@ -283,7 +283,7 @@ impl TextRun {
         let mut current: Option<(TextRunSegment, FontRef)> = None;
         let mut results = Vec::new();
 
-        let text = std::mem::replace(&mut self.text, String::new());
+        let text = std::mem::take(&mut self.text);
```

Comment:
This is an unrelated suggestion from clippy. It would be fine if you had to do other changes in this file, but otherwise I think it's better not to touch the file in this commit.

Reply:
Sorry about this, I thought i had removed these cahnges